content/docker/flask-and-docker/_index.md

```diff
@@ -9,12 +9,14 @@ prerequisites:
   - docker/intro-to-docker
   - projects/python-specific/flask-rest-api/part-1
   - projects/python-specific/flask-rest-api/part-2
+  soft:
```

Comment:
please remove the docker/docker-desktop-troubleshooting prerequisite. It should only be in the intro to docker content